### PR TITLE
feat: convert adding development redirect tests to Playwright

### PR DESCRIPTION
**Pull request by Sembauke**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
There's no need to check the response and try to find which url the browser is being redirected to. All we care about is which page the browser ends up at. i.e. `expect(page).toHaveUrl(expected)`

Given challenges.spec does this properly, I think it makes sense to consolidate the redirections into that file (maybe rename it redirects.spec.ts, too, since that's a better description).

**Review by huyenltnguyen — APPROVED** [review]

File: e2e/redirect.spec.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+// To run this test locally you will need to run: pnpm run start-ci;
+// Also, make sure that you have pm2 installed globally via: pnpm install -g pm2
+
+const pathsToTest = [
+  { input: '/challenges', expected: '/learn' },
+  {
+    input: '/learn/front-end-libraries/bootstrap',
+    expected: 'learn/front-end-development-libraries/bootstrap'
+  },
```
Comment (ojeytonwilliams) [inline e2e/redirect.spec.ts:11]:
```suggestion
    input: '/learn/front-end-libraries',
    expected: 'learn/front-end-development-libraries'
  },
  {
    input: '/learn/front-end-libraries/bootstrap',
    expected: 'learn/front-end-development-libraries/bootstrap'
  },
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Just one missing:

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Pull request merged**